CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 After upgrading, update your cache file by deleting it or via `tldextract
 --update`.
 
+## 2.1.1 (?)
```

Comment:
No need to fill out release details in this PR. I can take care of that when I release.